src/clients/github.js

```diff
@@ -25,8 +25,12 @@ function normalizeTitle(title) {
   return titleWithoutPunctuationAndWhitespace;
 }
 
+function getRepoNameForUser(project, user) {
+  return project.externalLocations.githubRepos[user.githubUsername];
```

Comment:
This function is basically the same as the selector `getCurrentProjectExportedRepoName` except we don't (and shouldn't) have access to the whole store here. Is there any literature on sharing these business-logicky things between redux and non-redux contexts